src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * Add `XmlEncoder::SAVE_OPTIONS` context option
  * Add `BackedEnumNormalizer::ALLOW_INVALID_VALUES` context option
  * Deprecate `MissingConstructorArgumentsException` in favor of `MissingConstructorArgumentException`
+ * Add option to use `Groups` attribute/annotation on class
```

Comment:
Is it necessary to support annotation + attribute instead of attribute only?

Reply:
@alexislefebvre  
No, it's not necessary. I added just because it already had support for both